dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/DependencyList.java

```diff
@@ -16,146 +16,89 @@
-                        }
-                        continue;
-                    }
-
-                    // scope
-                    String scope = null;
-                    NodeList sl = node.getElementsByTagName("scope");
-                    if (sl.getLength() > 0) {
-                        scope = sl.item(0).getTextContent();
-                    }
-                    if ("test".equals(scope) || "import".equals(scope)) {
-                        // skip test/BOM import scopes
-                        continue;
-                    }
-
-                    // version
-                    if (v == null && g.equals("org.apache.camel")) {
-                        v = camelVersion;
-                    }
-                    if (v == null && g.equals("org.apache.camel.springboot")) {
-                        v = camelVersion;
-                    }
-                    if (v == null && g.equals("org.springframework.boot")) {
-                        v = springBootVersion;
-                    }
-                    if (v == null && (g.equals("io.quarkus") || g.equals("org.apache.camel.quarkus"))) {
-                        v = quarkusVersion;
-                    }
-
-                    if (skipArtifact(g, a, v)) {
-                        continue;
-                    }
-                    if (v != null) {
-                        gavs.add(MavenGav.parseGav(g + ":" + a + ":" + v));
-                    } else {
-                        gavs.add(MavenGav.parseGav(g + ":" + a));
-                    }
-                }
-                // sort GAVs
-                gavs.sort(mavenGavComparator());
-                int i = 0;
-                int total = gavs.size();
-                for (MavenGav gav : gavs) {
-                    outputGav(gav, i, total);
-                    i++;
-                }
-            }
-            // cleanup dir after complete
-            File buildDir = new File(EXPORT_DIR);
-            FileUtil.removeDir(buildDir);
+    public void calculate() throws Exception {
+        List<String> routeFiles = new ArrayList<>();
+        for (Path filePath : filePaths) {
+            routeFiles.add("file://" + filePath.toAbsolutePath());
+        }
+        final KameletMain main = new KameletMain();
+        main.setDownload(false);
+        main.setFresh(false);
+        RunDownloadListener downloadListener = new RunDownloadListener();
+        main.setDownloadListener(downloadListener);
+        main.setSilent(!verbose);
+        // enable stub in silent mode so we do not use real components
+        main.setStubPattern("*");
+        // do not run for very long in silent run
+        main.addInitialProperty("camel.main.autoStartup", "false");
+        main.addInitialProperty("camel.main.durationMaxSeconds", "1");
+        main.addInitialProperty("camel.jbang.verbose", Boolean.toString(verbose));
+        main.addInitialProperty("camel.jbang.ignoreLoadingError", Boolean.toString(!verbose));
```

Comment:
Why `verbose` parameter is used to `ignoreLoadingError`?

Reply:
I fear it will break `dependency copy `on Camel routes which are in-progress. Which is a major use case when using in IDE integration (for reference https://github.com/camel-tooling/vscode-camelk/blob/b625bd537a52ca603edb0a92173dcd2292e48a37/src/JavaDependenciesManager.ts#L52 )

Reply:
It was a leftover from previous test. Just removed now.